woocommerce/woocommerce-android
# Issue/6623 connect select payment plugin screen to vm

Closes: #6623 #6828 and #6629 



### Description


This PR connects the ViewModel with the Select payment plugin screen when the user has both Stripe and WCPay plugins installed and active as part of the Make Cha Ching not war project. 


### Testing instructions

*This feature is behind the feature flag `IPP_SELECT_PAYMENT_GATEWAY` and will work only in debug build. 
**This also needs to have both Stripe and WCPay accounts correctly setup

1 - Select a store in the US that has both WooCommerce stripe and WCPay plugins properly set up
2 - Go to Menu - Settings - In-Person Payments
3 - Note if the payment gateway selection screen is correctly displayed
4 - Click on both payment buttons
5 - Make sure the selected button has a purple stroke and a purple checkmark while the unselected plugin has a grey stroke and no checkmark is displayed. 
6 - Click on confirm payment method button and make sure the app selected is saved in the app prefs
7 - Test other scenarios, like Stripe account not properly set up or WCPay set up incomplete and make sure you are taken to the correct error message.


### Images/gif


![Screen Capture on 2022-07-04 at 14-02-27](https://user-images.githubusercontent.com/30724184/177196131-0aa97a82-3efa-417b-90cd-38f3afae9aee.gif)
